modtools/moddable-gods.lua

```diff
@@ -90,13 +94,14 @@ godFig.id = df.global.hist_figure_next_id
 df.global.hist_figure_next_id = 1+df.global.hist_figure_next_id
 godFig.info = df.historical_figure_info:new()
 godFig.info.spheres = {new=true}
-godFig.info.secret = df.historical_figure_info.T_secret:new()
-
+godFig.info.known_info = {new=true}
```

Comment:
I would keep this as this, actually (I think it's clearer and more consistent with the rest of the script) - or is there a difference I'm missing?
```suggestion
godFig.info.known_info = df.historical_figure_info.T_known_info:new()
```